Orange/widgets/visualize/owfreeviz.py

```diff
@@ -139,14 +139,16 @@ class OWFreeViz(OWAnchorProjectionWidget, ConcurrentWidgetMixin):
     graph = settings.SettingProvider(OWFreeVizGraph)
 
     class Error(OWAnchorProjectionWidget.Error):
-        no_class_var = widget.Msg("Data has no target variable")
+        no_class_var = widget.Msg("Data must have a target variable.")
+        multiple_class_vars = widget.Msg(
+            "Data must have just a single target variable.")
```

Comment:
"Just" is likely redundant. I prefer:
- "Data must have a single target variable."
- "Data must have only one target variable."

Reply:
OK. I chose the first one, the shorter.